### ISSUE TITLE
Question: How to detect that deferred deep link check is completed, and no deep link data is there

### ISSUE THREAD

**Issue by amberv0** [AppsFlyerSDK/appsflyer-react-native-plugin#588]
Hello!
I just configured UDL with `react-native-appsflyer@6.15.1` and everything seems to be working smoothly so far. 
But I can't find a way to detect when the deferred deep link check is completed. 

Basically, I'm trying to implement a scenario where I'm showing a splash screen until I know there is no data coming from a deep link (currently it's `onDeepLink`). If there is some (deferred) deep link data, I'd show one screen, and a different screen otherwise. 

Currently I see no way to achieve what I want without some random setTimeout with a check that `onDeepLink` has been called, but it's not optimal since it can possible take a while if user has bad internet connection. 

Am I missing something?

Thank you!

**Comment by al-af** [comment]
Hi @amberv0,

Thank you for reaching out!

From what I understand, you’d like to use the splash screen phase to determine, based on the deep link data, whether it’s a first launch and redirect the user to the appropriate page accordingly.
Could you clarify where you’re initializing the SDK? Is the SDK being started during the splash screen or only after it?
For more tailored assistance with your use case, I recommend contacting support for detailed guidance.
